productivity-suite/app/fragments/news/vite.config.ts

```diff
@@ -0,0 +1,50 @@
+import solid from "solid-start/vite";
+import cloudflareWorkers from "solid-start-cloudflare-workers";
+import { defineConfig, PluginOption } from "vite";
+
+export function addDefaultFnExportToBundle(regex: RegExp): PluginOption {
```

Comment:
nit: I would maybe update the name of the function (like `wrapAndExportMount`), just to separate it from the other one? 🤔 (not a big deal, kinda just thinking out loud)